src/diff/props.js

```diff
@@ -11,12 +11,17 @@ import options from '../options';
  */
 export function diffProps(dom, newProps, oldProps, isSvg) {
 	for (let i in newProps) {
-		if (i!=='children' && i!=='key' && (!oldProps || oldProps[i]!=newProps[i])) {
+		if (i!=='children' && i!=='key' && i!=='value' && (!oldProps || oldProps[i]!=newProps[i])) {
 			setProperty(dom, i, newProps[i], oldProps[i], isSvg);
 		}
 	}
+
+	// `value` needs to be applied after an input's `min`, `max` and `step`
+	// attribute are set. See preact/#850
```

Comment:
Do you mean after an input's `min`, `max` and `step` _properties_ are set?

Reply:
ohh you got me haha